CDCgov/prime-reportstream
# update for DCipher

This PR: Update for DCipher ticket - 16388
Made modifications for Dcipher and fixed de-identifying for OBX-24

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. Load new receiver: hhs-protect.yml
2. Send HL7 Message as ignore.ignore-mars-otc
3. Verify message gets routed to hhs-protect and confirm file was identified as found in document- https://www.nibib.nih.gov/covid-19/radx-tech-program/mars/hl7-message-de-identification-for-sending-to-hhs-protect 

## Changes
- Added his-protect.yml
- Updated de-identification rules

## Checklist

### Testing
